File: fernanda/Problema0022/Problema0022.java
import java.util.Scanner;

public class Problema0022 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
        float custo, preco, quantidadeConvite;

        System.out.print("Digite o custo do espetáculo teatral: ");
        custo = Float.parseFloat(input.nextLine());
        System.out.print("Digite o preço do convite desse espetáculo: ");
        preco = Float.parseFloat(input.nextLine());

        quantidadeConvite = custo/preco;

        System.out.print("Devem ser vendidos "+ quantidadeConvite +" convites para que, pelo menos, o custo do espetáculo seja alcançado.");
    }
}

File: fernanda/Problema0022/Problema0022.js
const prompt = require("prompt-sync")({ sigint: true });
var custo = prompt("Digite o custo do espetáculo teatral: ")
var preco = prompt(("Digite o preço do convite desse espetáculo: "))

var quantidadeConvite = custo/preco

console.log("Devem ser vendidos "+ quantidadeConvite +" convites para que, pelo menos, o custo do espetáculo seja alcançado.")